File: 5 Kyu/MaximumSubarraySum.js
var maxSequence = function(arr){
    if(arr.length == 0) return 0
    let max =0
    for(let i = 0; i < arr.length; i++){
      let big = 0
      let next = 0 
      for(let j = i; j < arr.length; j++){
        next += arr[j]
        if (next > big) big = next
      }
      if (big > max) max = big
    }
    return max;
    }

    

File: 5 Kyu/MaximumSubarraySum.py
#https://en.wikipedia.org/wiki/Maximum_subarray_problem
def max_sequence(arr):
    maximum = 0
    sum = 0
    for i in arr:
        sum = max(0, sum +i)
        maximum = max(maximum, sum)
        print(sum,maximum)
    return maximum    